java/src/jmri/jmrit/logixng/actions/ExecuteDelayed.java

```diff
@@ -455,7 +455,7 @@ public void disconnected(FemaleSocket socket) {
         
         @Override
         public void execute() throws JmriException {
-            if (_socket != null) {
+            if ( _socket != null && conditionalNG != null ) {
```

Comment:
If `conditionalNG` is null, we have a runtime error. So I would like this to be changed to:
```
public void execute() throws JmriException {
    if (conditionalNG == null) throw new NullPointerException("conditionalNG is null");
    if (_socket != null) {
```
`_socket` is null if the user hasn't yet added anything to the socket. So that part is correct.